src/pages/management/configuration-api/index.mdx

```diff
@@ -2133,9 +2133,9 @@ For properties without an explicit value assignment, the method will return the
 |                 |                                                                                 |
 | --------------- | ------------------------------------------------------------------------------- |
 | Method URL      | `https://api.livechatinc.com/v3.4/configuration/action/list_license_properties` |
-| Required scopes | `properties.license.value--all:ro`*                                             |
+| Required scopes | `properties.license.value--all:rw`*                                             |
 
-__*)__ If the requester's Client ID doesn't match the namespace, retrieving license properties requires `properties.license.value--all:ro`. The scope allows to
+__*)__ The method doesn’t require any scopes if the requester’s client ID matches the namespace. If it doesn't, retrieving license properties requires `properties.license.value--all:rw`. The scope allows to
```

Comment:
Descriptions were paraphrased in all versions to make it more clear as there were suggestions that it was not clear enough.